app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,25 @@
+$govuk-assets-path: "/submissions/assets/lib/govuk-frontend/govuk/assets/";
+$hmrc-assets-path: "/submissions/assets/lib/hmrc-frontend/hmrc/";
```

Comment:
`submissions` is gform context. `eeitt-admin-frontend` should be used instead.

Reply:
sure, will update